### PR TITLE
kotlin java parity, init Realm as include file

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Staging links:

Quick start: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/moartabs/android/quick-start.html
- same content, refactored "Initialize Realm" into an includes file

Initialize the Realm Client: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/moartabs/android/init-realmclient.html
- Added "Initialize Realm" section that was already written for quick start, which contains details about where and why you need to run ``Realm.init()``
- Java/Kotlin tabbed examples where previously we only had Kotlin examples

Call a Function: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/moartabs/android/call-a-function.html
- Java/Kotlin tabbed examples where previously we only had Kotlin examples

### CODE REVIEW

File: source/android/init-realmclient.txt [hunk 1]
```diff
@@ -38,15 +59,32 @@ such as custom timeouts for connections, codecs used for MongoDB Data Access,
...
-.. code-block:: kotlin
+.. tabs-realm-languages::
+
+      .. tab::
+         :tabid: java
+
+         .. code-block:: java
+
+            String appID = "<your App ID>"; // replace this with your App ID
+            Realm.init(this); // context, typically an Application or Activity
```
Comment (cbush) [inline source/android/init-realmclient.txt:70]:
```suggestion
            Realm.init(this); // `this` is a Context, typically an Application or Activity
```

File: source/android/init-realmclient.txt [hunk 2]
```diff
@@ -38,15 +59,32 @@ such as custom timeouts for connections, codecs used for MongoDB Data Access,
...
+                              .requestTimeout(30, TimeUnit.SECONDS)
+                              .build());
+
+      .. tab::
+         :tabid: kotlin
+
+         .. code-block:: kotlin
+
+            val appID = "<your App ID>" // replace this with your App ID
+            Realm.init(this) // context, typically an Application or Activity
```
Comment (cbush) [inline source/android/init-realmclient.txt:82]:
```suggestion
            Realm.init(this) // `this` is a Context, typically an Application or Activity
```

File: source/includes/android-initialize-realm.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
+Initialize Realm
+----------------
+
+Before you can use {+service-short+} in your {+app+}, you must
```
Comment (cbush) [inline source/includes/android-initialize-realm.rst:4]: Technically, this is *not* the context-free meaning of {+app+}!
```suggestion
Before you can use {+service-short+} in your app, you must
```

File: source/includes/android-initialize-realm.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+application runs.
+
+.. tabs-realm-languages::
+
+   .. tab::
+      :tabid: java
+   
+      .. code-block:: java
+
+         Realm.init(this); // context, usually an Activity or Application
```
Comment (cbush) [inline source/includes/android-initialize-realm.rst:24]:
```suggestion
         Realm.init(this); // `this` is a Context, usually an Activity or Application
```

File: source/includes/android-initialize-realm.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+   
+   .. tab::
+      :tabid: kotlin
+
+      .. code-block:: kotlin
+   
+         Realm.init(this) // context, usually an Activity or Application
```
Comment (cbush) [inline source/includes/android-initialize-realm.rst:31]:
```suggestion
         Realm.init(this) // `this` is a Context, usually an Activity or Application
```

File: source/android/call-a-function.txt [hunk 6]
```diff
@@ -37,27 +37,64 @@ Call from a Client Application
...
-               result -> Log.v(TAG, "Sum value: ${result.get()}")
-         }
-      } else {
-            Log.e(TAG, it.error.toString())
-      }
-   }
+
+.. tabs-realm-languages::
+
+      .. tab::
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/android/call-a-function.txt:43]: quite, thanks for noticing!

**Pull request merged**